### DISCUSSION TITLE
I'mk receiving the message "[16:36:16] Could not check for updates (Permission denied)."

### DISCUSSION THREAD

**Discussion by Guba666** [pawelsalawa/sqlitestudio#4520] [General]
At the startup I'm receiving this message:
[16:36:16] Could not check for updates (Permission denied).
Why is that?
The application (rev. 3.3.3) is running with Admin privileges.

**Comment by pawelsalawa** [comment]
Apparently you declined network access to the application when Windows firewall prompted you for it.
SQLiteStudio uses network only to check for updates online.

If you don't care about updates, you can ignore the message. If you don't want this message to appear, you can disable attempts to check for updates in SQliteStudio configuration dialog.

If you want to re-grant the access, so that updates are actually being checked, here's an instruction: https://superuser.com/questions/1029761/i-accidentally-declined-when-windows-asked-me-if-i-should-allow-a-gaming-server

**Comment by Guba666** [comment]
Sorry for being so moronic.
A hadn't realized I had a firewall rule to block SQLStudio.
Thanks however for your kind reply.

Sergio

Il giorno sab 1 ott 2022 alle ore 21:45 Paweł Salawa <
***@***.***> ha scritto:
